changelog.d/1257.feature.md

```diff
@@ -0,0 +1,4 @@
+Revert platform specific directories on MacOS and Windows
```

Comment:
Should this be considered as `bugfix` instead of `feature`?

Reply:
As this significantly changes the installation behavior I opted for `feature`, for visibility in the changelog as well. But I don't feel strongly about this. Let me know what you think and I'll change it accordingly.

Reply:
Sounds reasonable 👍